### PR TITLE
Implement webpack and use it for Bulma CSS

### PR DESCRIPTION
**Pull request by Xpirix**
- #29

### CODE REVIEW

File: webpack.config.js [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+console.log(`Webpack mode: ${mode}`);
+
+module.exports = {
+  entry: './qgisfeedproject/static/js/index',
+  output: {
+    path: path.resolve('./qgisfeedproject/static/bundles'),
+    filename: "bundle.js"
+  },
+  plugins: [
+    new LiveReloadPlugin({appendScriptTag: true}),
```
Comment (dimasciput) [inline webpack.config.js:16]: @Xpirix this should only enabled in dev mode

File: qgisfeedproject/static/style/bulma.scss [hunk 2]
```diff
@@ -0,0 +1,167 @@
+@charset "utf-8";
```
Comment (dimasciput) [inline qgisfeedproject/static/style/bulma.scss:1]: @Xpirix, is this a customized Bulma style? Perhaps we should rename it to something different.

**Review by dimasciput — CHANGES_REQUESTED** [review]

File: qgisfeedproject/static/style/bulma.scss (hunk 2 above)
Comment (dimasciput) [reply qgisfeedproject/static/style/bulma.scss:1]: I mean the file name.

File: webpack.config.js (hunk 1 above)
Comment (Xpirix) [reply webpack.config.js]: @dimasciput Okay, I added the condition

File: qgisfeedproject/static/style/bulma.scss (hunk 2 above)
Comment (Xpirix) [reply qgisfeedproject/static/style/bulma.scss]: @dimasciput , Yes, I've renamed it to custom.bulma.scss

**Review by dimasciput — APPROVED** [review]

**Pull request merged**